### PR TITLE
[CI] Enable amp accuracy check for inductor cpu

### PR DESCRIPTION
**Pull request by DiweiSun**
This is to enable inductor AMP accuracy check for on CPU in CI workflow to capture issue early. Three suites are included: timms, huggingface as well as torchbench.

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @chenyang78 @kadeng @chauhang

### CODE REVIEW

**Comment by DiweiSun** [comment]
This PR is from #125748 , which is closed by @lezcano due to too many people tagged.

**Review by jgong5 — APPROVED** [review]

File: .github/workflows/inductor.yml [hunk 1]
```diff
@@ -171,6 +171,11 @@ jobs:
           { config: "cpu_inductor_timm_freezing", shard: 2, num_shards: 2, runner: "linux.12xlarge" },
           { config: "cpu_inductor_torchbench_freezing", shard: 1, num_shards: 2, runner: "linux.12xlarge" },
           { config: "cpu_inductor_torchbench_freezing", shard: 2, num_shards: 2, runner: "linux.12xlarge" },
+          { config: "cpu_inductor_huggingface_amp_freezing", shard: 1, num_shards: 1, runner: "linux.16xlarge.spr" },
+          { config: "cpu_inductor_timm_amp_freezing", shard: 1, num_shards: 2, runner: "linux.16xlarge.spr" },
+          { config: "cpu_inductor_timm_amp_freezing", shard: 2, num_shards: 2, runner: "linux.16xlarge.spr" },
+          { config: "cpu_inductor_torchbench_amp_freezing", shard: 1, num_shards: 2, runner: "linux.16xlarge.spr" },
+          { config: "cpu_inductor_torchbench_amp_freezing", shard: 2, num_shards: 2, runner: "linux.16xlarge.spr" },
```
Comment (desertfire) [inline .github/workflows/inductor.yml:178]: cc @huydhn , we have been growing this CPU test list quite a lot lately. I know CPU instances are usually cheaper, but still want to check with you if budget will be a concern.

**Review by desertfire — APPROVED** [review]

**Pull request closed without merging**